clusterDeployer.py

```diff
@@ -103,6 +110,58 @@ def setup_all_vms(h: host.Host, vms, iso_path, virsh_pool) -> list:
     return futures
 
 
+def configure_bridge(h) -> None:
+    cmd = "systemctl enable libvirtd"
+    run_cmd(h, cmd)
+    cmd = "systemctl start libvirtd"
+    run_cmd(h, cmd)
+
+    # stp must be disabled or it might conflict with default configuration of some physical switches
+    # 'bridge' section of network 'default' can't be updated => destroy and recreate
+    # check that default exists and contains stp=off
+    cmd = "virsh net-dumpxml default"
+    ret = h.run(cmd)
+
+    if "stp='off'" not in ret.out:
+        logger.info("=================== !!!!!!!!!!!!!!!!!!!!!! ====================")
+        logger.info(" !!!!!!!!!!!!! Destoying and recreating bridge !!!!!!!!!!!!!!!!")
+        logger.info("=================== !!!!!!!!!!!!!!!!!!!!!! ====================")
+        logger.info("\tcreating default-net.xml on localhost")
+        contents = """
+<network>
+  <name>default</name>
+  <forward mode='nat'/>
+  <bridge name='virbr0' stp='off' delay='0'/>
+  <ip address='192.168.122.1' netmask='255.255.0.0'>
+    <dhcp>
+      <range start='192.168.122.2' end='192.168.122.254'/>
+    </dhcp>
+  </ip>
+</network>
+"""
+        bridge_xml = os.path.join("/tmp", 'vir_bridge.xml')
+        with open(bridge_xml, 'w') as outfile:
+            outfile.write(contents)
+
+        cmd = "virsh net-destroy default"
+        h.run(cmd) # ignore return code - it might fail if net was not started
+
+        cmd = "virsh net-undefine default"
+        ret = h.run(cmd)
+        if ret.returncode != 0 and "Network not found" not in ret.err:
+            logger.error(ret)
+            sys.exit(-1)
+
+        cmd = f"virsh net-define {bridge_xml}"
+        run_cmd(h, cmd)
+
+        cmd = "virsh net-start default"
+        run_cmd(h, cmd)
```

Comment:
Again, I prefer to have a function `ensure_...` to ensure that the network is present